figma.html

```diff
@@ -0,0 +1,447 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="figma.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css"
+    />
+
+    <title>Trafalgar</title>
+  </head>
+  <body>
+    <nav class="navbar navbar-expand-lg py-3 navbar-light">
+      <div class="container">
+        <a href="#" class="navbar-brand">
+          <img src="img/image6.png" alt="Trafalgar Logo" class="me-2" width="30">
+          Trafalgar
+      </a>
+        <button
+          class="navbar-toggler"
+          type="button"
+          data-bs-toggle="collapse"
+          data-bs-target="#navmenu"
+        >
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navmenu">
+          <ul class="navbar-nav ms-auto">
+            <li class="nav-item">
+              <a href="" class="nav-link active">Home</a>
+            </li>
+            <li class="nav-item">
+              <a href="" class="nav-link">Find a doctor </a>
+            </li>
+            <li class="nav-item">
+              <a href="" class="nav-link">Apps</a>
+            </li>
+            <li class="nav-item">
+              <a href="" class="nav-link">Testimonials</a>
+            </li>
+            <li class="nav-item">
+              <a href="" class="nav-link">About us</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    <section class="pb-p-5 p-lg-0 text-center text-sm-start">
+      <div class="container">
+        <div class="d-flex align-items-center justify-conetent-center">
+          <div>
+            <h1>Virtual healthcare </h1>

```

Comment:
jedan h1 po stranici, ovo ti može biti h1 pa ispod h2 ili možeš to napraviti s h1 i spanom ako trebaš nešto pobojati drukčije na primjer ili ako ti se treba lomiti tekst iznimno to možeš s <br> tako da je u h1:
h1 Virtual healthcare br for you h1